File: test-suite-kotlin/src/test/kotlin/io/micronaut/docs/basics/Message.kt
package io.micronaut.docs.basics

// tag::imports[]

import com.fasterxml.jackson.annotation.JsonCreator
import com.fasterxml.jackson.annotation.JsonProperty

// end::imports[]

// tag::class[]
class Message @JsonCreator
constructor(@param:JsonProperty("text") val text: String)
// end::class[]

File: test-suite-python/src/test/python/micronaut/docs/basics/Message.py
from dataclasses import dataclass

# tag::imports[]
from micronaut.core.annotation import Introspected, ReflectiveAccess
# end::imports[]


# tag::class[]
@ReflectiveAccess
@Introspected
@dataclass
class Message:
    text: str | None = None
# end::class[]
